utils/utils.py

```diff
@@ -1,19 +1,25 @@
+import os
 from csv import DictReader
 
 from typing import Dict, List
 
+DEBUG = True
```

Comment:
Change back to False before final release

Reply:
Good catch!